docs/install/java_setup.md

```diff
@@ -85,6 +85,12 @@ Also, add the dependency which corresponds to your platform to the `dependencies
 The official Java Packages will be released with the release of MXNet 1.4 and will be available on  [MXNet Maven package repository](https://search.maven.org/#search%7Cga%7C1%7Cg%3A%22org.apache.mxnet%22).
 <hr>
 
+### Eclipse IDE Support
+You can easily convert you existing maven project to a project that can run in Eclipse by:
+```
+mvn eclipse:eclipse
```

Comment:
From what directory? Is this known or should you mention it?

Reply:
This is known, mvn command could only execute in a folder that has a pom file.

Reply:
I also add one line to explain that